cg/cli/upload/base.py

```diff
@@ -76,28 +79,28 @@ def upload(context, family_id, force_restart):
             click.echo(click.style(message, fg="yellow"))
             return
 
-    context.obj["report_api"] = ReportAPI(
-        store=analysis_api.status_db,
-        lims_api=analysis_api.lims_api,
-        chanjo_api=analysis_api.chanjo_api,
+    context.obj.meta_apis["report_api"] = ReportAPI(
+        store=status_db,
+        lims_api=context.obj.lims_api,
+        chanjo_api=context.obj.chanjo_api,
         analysis_api=analysis_api,
-        scout_api=analysis_api.scout_api,
+        scout_api=context.obj.scout_api,
     )
 
-    context.obj["scout_upload_api"] = UploadScoutAPI(
-        hk_api=analysis_api.housekeeper_api,
-        scout_api=analysis_api.scout_api,
-        madeline_api=analysis_api.madeline_api,
+    context.obj.meta_apis["scout_upload_api"] = UploadScoutAPI(
+        hk_api=context.obj.housekeeper_api,
+        scout_api=context.obj.scout_api,
+        madeline_api=context.obj.madeline_api,
         analysis_api=analysis_api,
-        lims_api=analysis_api.lims_api,
+        lims_api=context.obj.lims_api,
     )
 
     if context.invoked_subcommand is not None:
         return
 
     if not family_id:
         suggest_cases_to_upload(context)
-        context.abort()
+        raise click.Abort
```

Comment:
```suggestion
        return
```

Maybe just return? Or it will look like an error which might be not inteded

Reply:
I don't dare to mess with the API. Some tests/cronjobs/whatever might expect that an error is raised